### PR TITLE
Replace `args_to_dict`

### PR DESCRIPTION
**Pull request by cyrilrbt**

### CODE REVIEW

File: prf/utils/dictset.py [hunk 1]
```diff
@@ -700,54 +700,55 @@ def dict_to_args(d, keep_lists=False):
...
+        for _ in range(length - len(_list)):
+            _list.append({})
 
-    for arg in keys:
-        value = _args[arg]
 
-        bits = dot_split(arg)
-        ctx = _d
+def unflat(d):
+    r = {}
```
Comment (vahana) [inline prf/utils/dictset.py]: not a big fan of single char variables. this messes up with ipdb.

File: prf/utils/dictset.py [hunk 2]
```diff
@@ -534,11 +534,11 @@ def flat(self, keep_lists=True):
 
     def unflat(self):
         if hasattr(self, '_ordered_dict'):
-            _d = dictset(args_to_dict(self._ordered_dict))
+            _d = dictset(unflat(self._ordered_dict))
```
Comment (vahana) [inline prf/utils/dictset.py]: we should not need the _ordered_dict anymore as per new flat/unflat functions

Comment (cyrilrbt) [inline prf/utils/dictset.py]: I removed it in the next PR

File: prf/utils/dictset.py (hunk 1 above)
Comment (cyrilrbt) [inline prf/utils/dictset.py]: same, fixed in the next PR. I think it's easier to look at the next one and ignore this one altogether.

**Pull request closed without merging**